groups/ntc/ntcq/ntcq_send.h

```diff
@@ -797,6 +764,8 @@ bool SendQueue::popEntry()
     {
         SendQueueEntry& entry = d_entryList.front();
 
+        entry.closeTimer();
```

Comment:
The timer is forced to be closed in `popEntry`, but it is not "closed" in `popSize`. Why is there a difference?

Reply:
The closure of the timer is upon the first partial-write of data on the write queue is done by the caller, but for consistency and safety we should ensure the timer is closed in this function, as we do in `popEntry`.